arangod/Cluster/HeartbeatThread.cpp

```diff
@@ -697,6 +707,8 @@ void HeartbeatThread::getNewsFromAgencyForCoordinator() {
   if (++_DBServerUpdateCounter >= 60) {
     ci.loadCurrentDBServers();
     _DBServerUpdateCounter = 0;
+    auto& clusterFeature = server().getFeature<ClusterFeature>();
+    clusterFeature.pruneAsyncAgencyConnectionPool();
   }
 }
```

Comment:
Do we need to add the call to `pruneAsyncAgencyConnectionPool()` also in the `runSingleServer()` method of the HeartbeatThread? That method is used in active failover deployments, and it also uses the agency...

Reply:
yes

Reply:
We will address this in a second PR